packages/playground/interactive-block-playground/src/main.ts

```diff
@@ -0,0 +1,134 @@
+import {
+	connectPlayground,
+	installPlugin,
+	login,
+	PHPResponse,
+	PlaygroundClient,
+} from '@wp-playground/client';
+
+function asDOM(response: PHPResponse) {
+	return new DOMParser().parseFromString(response.text, 'text/html')!;
+}
+
+async function createNewPost(
```

Comment:
Follow-up: This would make a great Blueprint step

Reply:
Opened a separate PR for this: https://github.com/WordPress/wordpress-playground/pull/234